bx_django_utils_tests/tests/test_assert_queries.py

```diff
@@ -96,6 +99,46 @@ def test_assert_table_counts_exclude(self):
         msg = str(err.exception)
         assert msg == 'Excluded key \'auth_permission\' is listed in table_counts'
 
+    def test_snapshot_table_counts(self):
+        with AssertQueries() as queries:
+            Permission.objects.all().first()
+            Permission.objects.all().first()
+            Group.objects.all().first()
+
+        with AssertQueries() as second_queries:
+            Permission.objects.all().first()
+            Permission.objects.all().first()
+            Permission.objects.all().first()
+            Group.objects.all().first()
+
+        with tempfile.TemporaryDirectory() as tmp_dir:
```

Comment:
Think it's better to create a normal snapshot file and commit it. So we can see, how it looks like ;)